src/clipboard.ts

```diff
@@ -105,7 +105,9 @@ export async function copyTextToSystemClipboard(text: string | null) {
       //  not focused
       await navigator.clipboard.writeText(text || "");
       copied = true;
-    } catch (error) {}
+    } catch (error) {
+      console.error(error);
```

Comment:
Do we really want to ship these `console.log` to production?

Reply:
What's the alternative at this point? Better logging than nothing at all.

Reply:
At some point we’ll have Sentry to catch and log all the errors but this is what everywhere else in the codebase currently uses.